site/web/assets/js/classification.js

```diff
@@ -0,0 +1,139 @@
+var insertClassification = function (classificationDetails) {
+  var count = 1;
+  for (var i = 0; i < classificationDetails.length; i++) {
+    if (classificationDetails[i]["parentId"] === null) {
+      $('#category-' + count + '').append($("<option>").val(classificationDetails[i]["name"]).html(classificationDetails[i]["name"]));
```

Comment:
the `+ ''` is unnecessary. (Same a couple times below.)

And please stay with one style of string delimiters (' or ")

And break the line to get somewhere closer to 80 characters.